modules/local/chai_1/main.nf

```diff
@@ -7,25 +7,32 @@ process CHAI_1 {
     input:
     tuple val(meta), path(fasta)
     path weights_dir
+    val num_trunk_recycles
+    val num_diffn_timesteps
+    val seed
+    val use_esm_embeddings
 
     output:
     tuple val(meta), path("${meta.id}/*.cif"), emit: structures
     tuple val(meta), path("${meta.id}/*.npz"), emit: arrays
     path "versions.yml"                      , emit: versions
 
     script:
-    def downloads_dir = weights_dir ?: './downloads'
+    def args = task.ext.args ?: ''
+    def esm_flag = use_esm_embeddings ? '--use-esm-embeddings' : ''
     """
-    CHAI_DOWNLOADS_DIR=$downloads_dir \\
     run_chai_1.py \\
-        --output-dir ${meta.id} \\
-        --fasta-file ${fasta}
+        --fasta-file ${fasta} \\
+        --output-dir . \\
+        --num-trunk-recycles ${num_trunk_recycles} \\
+        --num-diffn-timesteps ${num_diffn_timesteps} \\
+        --seed ${seed} \\
+        ${esm_flag} \\
+        $args
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        python: \$(python --version | sed 's/Python //g')
```

Comment:
Why did you remove this @FloWuenne ?

Reply:
Don't remember removing this, must have been accidental, sorry!